matlab/test_image_reconstruction.m

```diff
@@ -48,7 +49,8 @@
 % Convert quaternions to rotation matrices
 rotmats_ctrl = zeros(3,3,num_poses_ctrl);
 for k = 1:num_poses_ctrl
-    rotmats_ctrl(:,:,k) = quat2rotm(quats(k,:));
+%     rotmats_ctrl(:,:,k) = quat2rotm(quats(k,:)); %Requires Robotic System Toolbox
+    rotmats_ctrl(:,:,k) = q2R(quats(k,:)); %Removes the need for Robotic System Toolbox
```

Comment:
Yes, I had noticed that they are not exact, but it did seem good enough. 

Regarding the toolboxes, explicit checking is definitely possible. There are similar (but different name) functions in one of the Simulink toolboxes, so it could check for that too if desired.